### PR TITLE
Add EX100 keyboard/mouse combo and support for 27 MHz receiver

### LINKED ISSUE #347: EX100 basic support intersted to anyone?
Anybody interested in support for EX100? Its rather old Logitech mouse/keyboard combo kit. 
http://support.logitech.com/en_us/product/cordless-desktop-ex-100-product
It is not Unifying, looking to receiver I'm think it's even operated in different RF frequency range, but, fortunately it support some limited subset of HID++ 1.0 commands. At least battery status reported correctly. 
Unfortunately, adding support for EX100 in Solaar require lots of hacks and workarounds in solaar code. I'm not very happy with code quality, but if it needed to someone except me i can try to create PR.

### PR DESCRIPTION
**Pull request by doctor64**
docs: add EX100 keyboard/mouse combo
receiver: add EX100 27 MHz receiver (046d:c517), removed incorrectly 
listed as nano receiver, add workarounds for EX100 wpids
device: add EX100 keyboard and mouse
hidpp10: fix notification parsing, add device features flags
notifications: fix wpid processing for 27 MHz protocol
cli: add display of device features flags for HID++ 1.0 devices

Fixes #347

### CODE REVIEW

File: lib/logitech_receiver/hidpp10.py [hunk 1]
```diff
@@ -116,6 +128,25 @@
     notifications=0x00,
     firmware=0xF1,
 )
+# Flags taken from https://drive.google.com/drive/folders/0BxbRzx7vEV7eWmgwazJ3NUFfQ28
```
Comment (Klap-in) [inline lib/logitech_receiver/hidpp10.py:131]: Hi,
This is the link to the folder, it might be handy to mention which file you used from this folder?

Comment (doctor64) [reply lib/logitech_receiver/hidpp10.py]: Hi,
links updated.

**Comment by pfps** [comment]
I'm working through the PR.   One change to make - the commit name needs to be changed to something like "receiver: add support for EX100 keyboard/mouse and receiver"

**Comment by pfps** [comment]
@FFY00 Something different appears to be happening when the master branch is advanced.  I thought that it used to be that PRs could be merged nicely even after another PR was added to the master branch.  But now there is this "out-of-date" message and the proposed remedy appears to add a merge commit. Do you know what is goiing on?

**Comment by doctor64** [comment]
@pfps 

> The only change I recommend is to change the commit name.

Do you mean PR name?

**Comment by pfps** [comment]
@doctor64  No, the commit name, which right now is "docs: ...".  You can change that locally and force-push the change.

**Comment by pfps** [comment]
@doctor64 I think that you pushed the button that purports to merge changes on the main branch into your branch.  I did that too, but it does a merge into your branch, which may end up putting a merge commit into Solaar, which is not a good idea.   I think the way to proceed is to re-push your commit(s), but @FFY00 may have better guidance.

**Comment by FFY00** [comment]
I just tested and Github will rebase away the merge commit. See #860 and https://github.com/pwr-Solaar/Solaar/commits/test. However this is really confusing, when you click the "Update branch" button Github should rebase instead of creating a merge commit. Github is once again showing us it is designed by people who don't care about git history. Hey, at least they now warn us when a branch is out of date with master, I always thought it was crazy they didn't.

Go ahead and click the button.

Anyway, I have a branch rule that should prevent merge commits from being merged/pushed to master, so I think you don't need to worry. If something bad happens, I won't scream at you, I will scream at Github.

**Comment by pfps** [comment]
OK, pushing the button.

**Comment by pfps** [comment]
@FFY00  Yes!  The right thing happened.

**Comment by pfps** [comment]
@doctor64 It appears that EX100s do report full WPIDs in connection notifications.  Your PR assumed that they only reported one byte of the WPID.  PR #932 removes this special handling.  Please try it out and see whether Solaar does the right thing for you.   See also the last part of issue #716

**Pull request merged**